common/build/eslint-config-fluid/src/test/enforce-no-restricted-tags-imports/enforce-no-restricted-tags-imports.test.js

```diff
@@ -28,11 +28,11 @@ describe("ESLint Rule Tests", function () {
 			},
 			parser: "@typescript-eslint/parser",
 			parserOptions: {
-				project: path.join(__dirname, "./tsconfig.json"),
+				project: path.join(__dirname, "../tsconfig.json"),
 			},
 		});
 		const filesToLint = ["fileWithImports.ts", "mockModule.ts"].map((file) =>
-			path.join(__dirname, "mockFiles", file),
+			path.join(__dirname, ".././mockFiles/no-restricted-tags-imports", file),
```

Comment:
Let's apply this on the two places below that did the same thing.
```suggestion
			path.join(__dirname, "../mockFiles/no-restricted-tags-imports", file),
```